pkg/sentry/platform/kvm/bluepill_unsafe.go

```diff
@@ -48,166 +46,3 @@ func bytePtr(addr uintptr) *byte {
 func uintptrValue(addr *byte) uintptr {
 	return (uintptr)(unsafe.Pointer(addr))
 }
-
-// bluepillHandler is called from the signal stub.
-//
-// The world may be stopped while this is executing, and it executes on the
-// signal stack. It should only execute raw system calls and functions that are
-// explicitly marked go:nosplit.
-//
-//go:nosplit
-func bluepillHandler(context unsafe.Pointer) {
```

Comment:
why do you move bluepillHandler to an arch-specific file? I think it is better to do in a separate pr.